.buildkite/pipeline.yml

```diff
@@ -131,6 +136,36 @@ steps:
     concurrency_group: 'bitbar'
     concurrency_method: eager
 
+  - label: ':browserstack: Android 7 NDK r19 ANR smoke tests'
+    depends_on: "fixture-r19"
+    timeout_in_minutes: 60
+    plugins:
+      artifacts#v1.9.0:
+        download:
+          - "build/bs-fixture-r19-url.txt"
+          - "build/fixture-r19/*"
+        upload:
+          - "maze_output/failed/**/*"
+          - "maze_output/metrics.csv"
+      docker-compose#v4.7.0:
+        pull: maze-runner
+        run: maze-runner
+        service-ports: true
+        command:
+          - "features/smoke_tests/01_anr.feature"
+          - "--app=@build/bs-fixture-r19-url.txt"
+          - "--app-activity=com.bugsnag.android.mazerunner.MainActivity"
+          - "--app-package=com.bugsnag.android.mazerunner"
+          - "--appium-version=1.22.0"
+          - "--farm=bs"
+          - "--device=ANDROID_7"
```

Comment:
An annoying thing to realise now is that all the android 7 and 8 devices we have for BS in Maze Runner are Samsungs and so the scenarios will all be skipped.

Maybe we don't actually need to skip Samsung?  Could be worth checking that.  Or see if BS have any other options we can use?

We can skip the Android 7 and 8 steps in this PR if you like and raise a separate ticket to reinstate them somehow.

Reply:
Looking at BS it doesn't look like they have any none Samsung devices for Android 7/8. I'll remove the skip and see what issues we get.